packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -345,32 +348,39 @@ export function makeRumPublicApi(
          * @param options.startTime epoch timestamp of the start of the custom vital
          * @param options.duration duration of the custom vital
          * @param options.context custom context attached to the vital
-         * @param options.details  Details of the vital.
+         * @param options.description  Descritpion of the vital
          */
         ;(rumPublicApi as any).addDurationVital = monitor(
-          (name: string, options: { startTime: number; duration: number; context?: object; details?: string }) => {
+          (name: string, options: { startTime: number; duration: number; context?: object; description?: string }) => {
             addTelemetryUsage({ feature: 'add-duration-vital' })
             strategy.addDurationVital({
               name: sanitize(name)!,
               type: VitalType.DURATION,
               startClocks: timeStampToClocks(options.startTime as TimeStamp),
               duration: options.duration as Duration,
               context: sanitize(options && options.context) as Context,
-              details: sanitize(options && options.details) as string | undefined,
+              description: sanitize(options && options.description) as string | undefined,
             })
           }
         )
 
         /**
-         * @deprecated
          * Stop a custom duration vital
          * stored in @vital.custom.<name>
          *
-         * @param name name of the custom vital
+         * @param nameOrRef name of the custom vital or the reference to the custom vital
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ I'd like to remove the duplicate _custom vital_ in that sentence, but I'm not convince it's better :(
```suggestion
         * @param nameOrRef name of the custom vital or the reference to it
```